### PR TITLE
Fixed typo signalStrength

### PR DESCRIPTION
**Pull request by khanhdodang**
## Proposed changes

Fixed typo `signalStrength`

## Types of changes

[//]: # (What types of changes does your code introduce to WebdriverIO?)
[//]: # (_Put an `x` in the boxes that apply_)

- [x] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist

[//]: # (_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code._)

- [x] I have read the [CONTRIBUTING](https://github.com/webdriverio/webdriverio/blob/master/CONTRIBUTING.md) doc
- [x] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)

## Further comments

[//]: # (If this is a relatively large or complex change, kick off the discussion by explaining why you chose the solution you did and what alternatives you considered, etc...)

### CODE REVIEW

**Comment by khanhdodang** [comment]
I also added PR for appium-base-driver too https://github.com/appium/appium-base-driver/pull/308

**Comment by christian-bromann** [comment]
Ok, how do we ensure backwards compatibility?

**Comment by khanhdodang** [comment]
Hi @christian-bromann @mgrybyk I updated it from appium-base-driver too. Please see the PR here https://github.com/appium/appium-base-driver/pull/308.
I also fixed for other clients too.

File: packages/webdriver/protocol/appium.json [hunk 1]
```diff
@@ -944,10 +944,13 @@
       "description": "Set GSM signal strength (Emulator only).",
       "ref": "http://appium.io/docs/en/commands/device/network/gsm-signal/",
       "parameters": [{
+        "name": "signalStrength",
+        "type": "string",
+        "description": "signal strength in the range [0, 4]"
```
Comment (christian-bromann) [inline packages/webdriver/protocol/appium.json]: the `signalStrength` parameter should be required and in the description it should say that this parameter works with Appium vX upwards and the other has to be set if user is using a lower version

Comment (khanhdodang) [reply packages/webdriver/protocol/appium.json]: Thanks @christian-bromann I addressed your comment.

Comment (khanhdodang) [reply packages/webdriver/protocol/appium.json]: @christian-bromann The PR https://github.com/appium/appium-base-driver/pull/308 has been merged.

File: packages/webdriver/protocol/appium.json [hunk 2]
```diff
@@ -944,10 +944,14 @@
...
-        "name": "signalStrengh",
+        "name": "signalStrength",
         "type": "string",
-        "description": "signal strength in the range [0, 4]",
+        "description": "signal strength in the range [0, 4]. This parameter works with Appium 1.11.1 upwards.",
         "required": true
+      }, {
+        "name": "signalStrengh",
+        "type": "string",
+        "description": "signal strength in the range [0, 4]"
```
Comment (christian-bromann) [inline packages/webdriver/protocol/appium.json]:
```suggestion
        "description": "signal strength in the range [0, 4]. Please also set this parameter with the same value if you use Appium v1.11.0 or lower (see https://github.com/appium/appium/issues/12234)."
```

Let's be 100% sure and add this comment.

Comment (khanhdodang) [reply packages/webdriver/protocol/appium.json]: Hi @christian-bromann The message now is exactly as your expectation.

Comment (christian-bromann) [reply packages/webdriver/protocol/appium.json]: @khanhdodang this description should be for the `signalStrengh` parameter .. the other can stay at it was before

Comment (khanhdodang) [reply packages/webdriver/protocol/appium.json]: Hi @christian-bromann I updated

**Pull request merged**